bin/alluxio

```diff
@@ -8,9 +8,12 @@ fi
 BIN=$(cd "$( dirname "$0" )"; pwd)
 
 function printUsage {
-  echo "Usage: alluxio COMMAND "
-  echo "where COMMAND is one of:"
-  echo -e "  format [-s]   \t Format Alluxio (if -s specified, only format if underfs is local and doesn't already exist)"
+  echo "Usage: alluxio COMMAND [GENERIC_OPTIONS] [COMMAND_OPTIONS]"
```

Comment:
Could we combine this into just [OPTIONS]? I've never heard of a tool with two types of options which need to be in a specific order